src/cli.ts

```diff
@@ -50,10 +51,21 @@ const logToolbeltVersion = () => {
   log.debug(`Toolbelt version: ${pkg.version}`)
 }
 
+const validToken = (): boolean => {
```

Comment:
**nit**: `tokenIsValid` or `hasValidToken`?

Reply:
True that